### PR TITLE
free global mutex in Gles3Context

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@emilwestergren review

### CODE REVIEW

File: source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc [hunk 1]
```diff
@@ -231,4 +231,6 @@ Gles3Context: class extends GLContext {
 		result _generate(sharedContext) ? result : null
 	}
 }
+
+GlobalCleanup register(|| Gles3Context _mutex free(), 10)
```
Comment (sebastianbaginski) [inline source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc]: `10` so it will run at the end.

Comment (marcusnaslund) [inline source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc]: But will it though :shipit:

**Comment by marcusnaslund** [comment]
Please restart print here

**Comment by sebastianbaginski** [comment]
I'm gonna even rebase.

**Comment by marcusnaslund** [comment]
Woah

**Pull request merged**